esp-radio/src/esp_now/mod.rs

```diff
@@ -24,10 +24,9 @@ use portable_atomic::{AtomicBool, AtomicU8, Ordering};
 use super::*;
 #[cfg(feature = "csi")]
 use crate::wifi::CsiConfig;
-use crate::{
-    binary::include::*,
-    wifi::{RxControlInfo, WifiError},
-};
+#[cfg(all(any(feature = "esp-now", feature = "sniffer"), feature = "unstable"))]
```

Comment:
if this file gets compiled the feature esp-now is active - we can remove the cfg-gate here